test/acceptance/libp2p/host/peer/test_peer.cpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "acceptance/libp2p/host/peer/test_peer.hpp"
+
+#include "acceptance/libp2p/host/protocol/client_test_session.hpp"
+
+Peer::Peer(Peer::Duration timeout)
```

Comment:
Please, add more comments to this class. It's hard to understand, what all of this is supposed to do: ticks, clients, servers...

Reply:
done